content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -222,7 +222,7 @@ val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
 
 ### Start the recording manually
```

Comment:
```suggestion
### Start or stop the recording manually
```

Reply:
I think adding this in the header will be good for anyone searching for this info.

Reply:
Agreed! Thanks for the quick review!